brainscore_vision/models/resnet50_robust_l2_eps1/test.py

```diff
@@ -0,0 +1 @@
+# Left empty as part of 2023 models migration
```

Comment:
For all converted models, we want a simple test to make sure the model is added correctly to Brain-Score:
```

import pytest
import brainscore_vision



@pytest.mark.travis_slow
def test_has_identifier():
    model = brainscore_vision.load_model('resnet50_robust_l2_eps1')
    assert model.identifier == 'resnet50_robust_l2_eps1'
```

This will allow(once the model is merged) Brain-Score to test and ensure that the model is located where it should be. You can also delete the comment on line 1.